client/README.md

```diff
@@ -25,7 +25,9 @@ export API_URL=https://api-HASH-REGION.a.run.app/
 export API_URL=http://localhost:8000
 ```
 
-Install, build, and start client.
+### Install, build, and start client locally
+
+Once you have the API server process running locally, in another terminal tab, execute the following:
```

Comment:
the above documented variable export API_URL=https://api-HASH-REGION.a.run.app/
implies that you don't have to have the API server process running locally, do you? is the key that it should be running at all or that it's local?

Reply:
Just as long as the API is running locally or deployed, client should be able to run locally if passed the env var `API_URL`.